src/content/reference/react/Fragment.md

```diff
@@ -111,15 +111,15 @@ function Post() {
 }
 ```
 
-Usually you won't need this unless you need to [pass a `key` to your `Fragment`.](#rendering-a-list-of-fragments)
+Biasanya Anda tidak memerlukan cara ini kecuali Anda perlu [mengoper `key` ke `Fragment` Anda.](#rendering-a-list-of-fragments)
 
 </DeepDive>
 
 ---
 
-### Assigning multiple elements to a variable {/*assigning-multiple-elements-to-a-variable*/}
+### Menentukan banyak elemen ke variabel {/*assigning-multiple-elements-to-a-variable*/}
```

Comment:
Assigning=Menetapkan?

cc: @mhaidarhanif @r17x @zainfathoni @resir014 

```suggestion
### Menetapkan banyak elemen ke variabel {/*assigning-multiple-elements-to-a-variable*/}
```

Reply:
Yes `menetapkan` is fine. Alternatively, `menugaskan`

Reply:
Hi @mazipan , I already change the word to `menetapkan`. Please review it again.

Reply:
The changes are still not applied yet.

![image](https://user-images.githubusercontent.com/6315466/236866805-b47d5d4b-135a-4cc3-8b0f-b1d0577aeffd.png)